### PR TITLE
Add event page, hook and components

### PR DESCRIPTION
**Pull request by Arashfa0301**
- added event component for the event page (EventItem, EventList)
- added event page
- added event hook that fetches events from Lego and can be easily reused 
- the design could be much better and is currently being fixed

### CODE REVIEW

File: components/Admissions/EventItem.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    party: "Fest",
+    social: "Sosialt",
+    event: "Arrangement",
+    kid_event: "KID-arrangement",
+    other: "Annet",
+  };
+
+  const eventTypeToString = (eventT: string) => {
+    return EVENT_CONSTANTS[eventT] || EVENT_CONSTANTS.other;
+  };
```
Comment (AdrianAndersen) [inline components/Admissions/EventItem.tsx]: These should be moved outside of the EventItem component, preferably inside a types.ts file somewhere in the repo. I suggest adding a folder called /lib/types.ts (in the repo root folder)

File: components/Admissions/EventItem.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+          textAlign="center"
+        >
+          {title}
+        </Text>
+        <Text mt="2" fontSize="sm" textAlign="center" color="coolGray.700">
+          {`${moment(startTime)
+            .utc()
+            .format("MMM.Do.HH:mm")} ⊙ ${location} ⊙ ${eventTypeToString(
+            eventType
+          )}`}
```
Comment (AdrianAndersen) [inline components/Admissions/EventItem.tsx]: Why is this UTC? Should it not be the current time zone?

File: components/Hooks/useEvents.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  endTime: string;
+  thumbnail: string;
+  totalCapacity: number;
+  company: Object;
+  registrationCount: number;
+  activationTime: string;
+  isAdmitted: boolean;
+  survey: string;
+  requireAuth: boolean;
+}
```
Comment (AdrianAndersen) [inline components/Hooks/useEvents.ts]: Could also be moved to a separate types.ts file as mentioned above

**Review by AdrianAndersen — CHANGES_REQUESTED** [review]

File: components/Admissions/EventItem.tsx (hunk 1 above)
Comment (Arashfa0301) [reply components/Admissions/EventItem.tsx]: I fixed. Thanks for the feedback.

File: components/Hooks/useEvents.ts (hunk 3 above)
Comment (Arashfa0301) [reply components/Hooks/useEvents.ts]: I fixed.

File: components/Admissions/EventItem.tsx (hunk 2 above)
Comment (Arashfa0301) [reply components/Admissions/EventItem.tsx]: I fixed.

**Comment by AdrianAndersen** [comment]
LGTM 🚀

**Review by AdrianAndersen — APPROVED** [review]
LGTM 🚀

File: package.json [hunk 4]
```diff
@@ -15,10 +15,12 @@
     "@expo/vector-icons": "^13.0.0",
     "@react-navigation/bottom-tabs": "^6.4.0",
     "@react-navigation/native": "^6.0.13",
+    "@tanstack/react-query": "^4.10.1",
     "expo": "~46.0.9",
     "expo-device": "^4.3.0",
     "expo-notifications": "~0.16.1",
     "expo-status-bar": "~1.4.0",
+    "moment": "^2.29.4",
```
Comment (LudvigHz) [inline package.json:23]: no-god

Comment (ivarnakken) [reply package.json:23]: 😬

Comment (LudvigHz) [reply package.json:23]: On a more serious note, moment is deprecated.
I suggest you use one of these (in my order of preference):
1. https://day.js.org/
2. https://moment.github.io/luxon/#/?id=luxon

**Pull request merged**